studio/schemas/deskStructure.ts

```diff
@@ -51,11 +51,11 @@ export default (S: StructureBuilder) =>
         .icon(ProjectsIcon)
         .child(S.documentTypeList(pageBuilderID).title("Dynamic Pages")),
       S.listItem()
-        .title("Blog")
-        .icon(ComposeIcon)
+        .title("Static Pages")
+        .icon(ProjectsIcon)
```

Comment:
Should we maybe use a different icon to differenciate between static and dynamic pages? What about `TiersIcon` or `StackCompactIcon` / `StackIcon`?

Reply:
Awesome. Can we also update the description with the new image? or add as a comment?